internal/transformer/classes/analysers/springbootanalyser.go

```diff
@@ -88,104 +107,400 @@ func (t *SpringbootAnalyser) BaseDirectoryDetect(dir string) (namedServices map[
 	return nil, nil, nil
 }
 
-// DirectoryDetect runs detect in each sub directory
-func (t *SpringbootAnalyser) DirectoryDetect(dir string) (namedServices map[string]transformertypes.ServicePlan, unnamedServices []transformertypes.TransformerPlan, err error) {
-	destEntries, err := ioutil.ReadDir(dir)
+// GetGradleData extracts info from Gradle files
+func GetGradleData(buildGradlePath string, settingsGradlePath string) (configuration Configuration, err error) {
```

Comment:
Right, for the moment I changed to lowercase. If we need to have multiple java transformers in the future, we can take it outside so it can consumable from different sources